src/styles/blocks/product-catalog.scss

```diff
@@ -0,0 +1,29 @@
+.product-catalog {
+  padding-block: 50px;
+  padding-inline: 40px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  grid-template-rows: repeat(auto-fill, 1fr);
```

Comment:
For `grid-template-rows`, using `auto-fill` is not valid. Instead, you should use `auto-fit` if you want the grid to create as many rows as needed, or simply omit this property if you don't need to set the number of rows.